pages/common/gist.md

```diff
@@ -0,0 +1,23 @@
+# gist
+
+> Upload code to https://gist.github.com.
+
+- Upload any number of files:
```

Comment:
These can only be text files, right? Would "Upload text files and create a gist out of them" be accurate, or does is a separate gist created for each file?